bentoml/deployment/sagemaker/__init__.py

```diff
@@ -165,8 +165,6 @@ def create_push_docker_image_to_ecr(bento_name, bento_version, snapshot_path):
     except ecr_client.exceptions.RepositoryNotFoundException:
         ecr_client.create_repository(repositoryName=image_name)
 
-    if docker_api.tag(image_name, ecr_tag) is False:
```

Comment:
why was this check here before?

Reply:
its not on the latest master, will rebase from that